airflow/utils/json.py

```diff
@@ -17,11 +17,12 @@
 # under the License.
 from __future__ import annotations
 
+import json
```

Comment:
At some point in the past, if user had simplejson installed, then webserver would blow up, that's why we imported from `flask.json`.  It might be worth doing pip install simplejson just to make sure that it is compatible

Reply:
it was this commit: https://github.com/apache/airflow/commit/ea3d42a3b68f926ff5022e2786bd6c57e3308cd2

Reply:
Tested that, doesn't blow up (by which I mean tests still create the app and pass)

Reply:
cool -- i just tried it too, installed it and launched webserver and navigated to a view  or two